db/migrate/20180412124505_add_priority_score_to_discovery_rules.rb

```diff
@@ -0,0 +1,23 @@
+ class AddPriorityScoreToDiscoveryRules < ActiveRecord::Migration[4.2]
+  class DiscoveryRules < ActiveRecord::Base
+  end
+
+  def up
+    score = DiscoveryRules.distinct.order("priority asc").pluck(:priority).first
+    # To keep the same priority behavior before this change,
+    # set the priority score accordingly.
+    DiscoveryRules.reset_column_information
+    rules = DiscoveryRules.order("priority asc, name asc")
+    rules.each do |rule|
+      rule.update_attributes!(:priority => score)
+      score += 1
```

Comment:
Can you simplify that and set initial score to `100` and then add `100` every loop so there is enough room for additional entries? I think we can ignore current priority numbers totally, they are not important. What's important is the order we must kept.